tests/contrib/django/django_tests.py

```diff
@@ -200,6 +201,25 @@ def test_user_info(self):
         self.assertTrue('email' in user_info)
         self.assertEquals(user_info['email'], 'admin@example.com')
 
+    def test_user_info_raises_operational_error(self):
+        user = User(username='admin', email='admin@example.com')
+        user.set_password('admin')
+        user.save()
+
+        self.assertTrue(
```

Comment:
this being pytest, you could just do `assert self.client ...`, and `assert x == y` instead of `self.assertEquals`

but not really important :)